scripts/gceworker.sh

```diff
@@ -168,6 +168,17 @@ case "${cmd}" in
     shift
     gcloud compute scp --recurse "${from}" "$@"
     ;;
+    put)
+    # scp allows one or more sources, followed by a single target.
+    if (( $# < 2 )); then
+        echo "usage: $0 put sourcepath [sourcepath...] targetpath"
```

Comment:
Nit: you could add a special case where if only 1 argument is provided, then the destination is assumed to be the home directory, as in `scripts/gceworker.sh put file` would put the `file` in `~`. Up to you.

Reply:
Good call! Done.

```
# michae2@michae2-crl-m3 146 (1) ~crdb % scripts/gceworker.sh put bar.diff
bar.diff                                                                                                                                                                                                                                                                                                                                                                                                  100% 4499   278.8KB/s   00:00

# michae2@michae2-crl-m3 (1) ~crdb % scripts/gceworker.sh ssh
Welcome to Ubuntu 20.04.6 LTS (GNU/Linux 5.15.0-1060-gcp x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/pro

 System information as of Wed Jun  5 22:27:08 UTC 2024

  System load:  0.0                Processes:             290
  Usage of /:   70.9% of 96.73GB   Users logged in:       1
  Memory usage: 15%                IPv4 address for ens4: 10.138.0.12
  Swap usage:   0%

 * Strictly confined Kubernetes makes edge and IoT secure. Learn how MicroK8s
   just raised the bar for easy, resilient and secure K8s cluster deployment.

   https://ubuntu.com/engage/secure-kubernetes-at-the-edge

Expanded Security Maintenance for Applications is not enabled.

0 updates can be applied immediately.

Enable ESM Apps to receive additional future security updates.
See https://ubuntu.com/esm or run: sudo pro status

New release '22.04.3 LTS' available.
Run 'do-release-upgrade' to upgrade to it.


Last login: Wed Jun  5 21:56:11 2024 from 97.126.89.218

michae2@gceworker-michae2:~$ ls
103714  103820  103869  106884  118273  121544  125115  2863  95264  bar.diff  baz.diff  bootstrap  foo.diff  go  local  old_crdbs  snap
```